packages/pigeon/lib/swift_generator.dart

```diff
@@ -263,7 +284,99 @@ class SwiftGenerator extends StructuredGenerator<SwiftOptions> {
         _writeClassField(indent, field, addNil: !classDefinition.isSwiftClass);
         indent.newln();
       }
+    });
+  }
 
+  void _writeCodecOverflowUtilities(
+    SwiftOptions generatorOptions,
+    Root root,
+    Indent indent,
+    List<EnumeratedType> types, {
+    required String dartPackageName,
+  }) {
+    final NamedType overflowInt = NamedType(
+        name: 'type',
+        type: const TypeDeclaration(baseName: 'Int', isNullable: false));
+    final NamedType overflowObject = NamedType(
+        name: 'wrapped',
+        type: const TypeDeclaration(baseName: 'Object', isNullable: true));
+    final List<NamedType> overflowFields = <NamedType>[
+      overflowInt,
+      overflowObject,
+    ];
+    final Class overflowClass =
+        Class(name: _overflowClassName, fields: overflowFields);
+    indent.newln();
+    indent.writeln('// swift-format-ignore: TypeNamesShouldBeCapitalized');
+    _writeDataClassSignature(indent, overflowClass);
```

Comment:
Can this be private (by adding a `{bool private = false}` param to add the private keyword)?